contents/handbook/growth/marketing/index.md

```diff
@@ -4,20 +4,29 @@ sidebar: Handbook
 showTitle: true
 ---
 
-The Marketing & Content team exist to drive organic, word-of-mouth growth by:
-
-1. Producing world-class top-of-funnel content for our ICP
-2. Ensuring the information users need is easy to find and maintained to a high standard
-
-Marketing is the [newsroom](https://en.wikipedia.org/wiki/Newsroom) of PostHog. This means:
-
-- A small, talented team of in-house content creators with strong editorial capability – our default is to keep writing in-house, not farm it out to a network of freelancers.
- 
-- Building a reputation for high-quality, trustworthy content that is a go-to resource for our users and wider reading audience. 
-
-- Extreme responsiveness to opportunities, both internal and external (e.g. cool Hackathon projects, GDPR rulings).
- 
-- Proactively reporting and amplifying company and product news, rather than [artificially creating news](https://en.wikipedia.org/wiki/Tomorrow_Never_Dies) or buzz by telling people what to build.
+The marketing team has two core goals:
+
+1. Increase awareness of PostHog, especially among people in our ideal customer profile.
+2. Make engineers and PostHog users feel like badass ninjas by teaching them cool stuff.
```

Comment:
> badass ninjas

A bit cringe in framing, IMHO. It's like posting a job ad asking for a rockstar.

Reply:
yeah, I figured the same tbh. I'll come up something else.

Reply:
"flex their product muscle" (this is how I frame Product for Engineers)